janitor/io.py

```diff
@@ -110,3 +111,66 @@ def read_commandline(cmd: str, **kwargs) -> pd.DataFrame:
     else:
         outcome = outcome.stdout
     return pd.read_csv(StringIO(outcome), **kwargs)
+
+
+def xlsx_table(
+    path: str,
+    sheetname: str,
+    table: Union[str, list, tuple] = None,
+    header: bool = True,
+) -> pd.DataFrame:
+    """
+    Returns a DataFrame of values in a table in the Excel file.
+    If the `table` argument is provided, a pandas DataFrame is returned;
+    if the `table` argument is None, or a list/tuple of names,
+    a dictionary of DataFrames is returned, where the keys of the dictionary
+    are the table names.
+
```

Comment:
Could you add in an example usage (using the triple back-ticks (```) , rather than the triple right-chevron `>>>`? 

We're avoiding the latter here because the I/O module can't be easily doc-tested, I think.